packages/flutter_tools/test/integration/test_driver.dart

```diff
@@ -188,9 +191,13 @@ class FlutterTestDriver {
     return _proc.exitCode;
   }
 
-  Future<void> addBreakpoint(String path, int line) async {
+  Future<VMIsolate> getFlutterIsolate() async {
     final VM vm = await vmService.getVM();
-    final VMIsolate isolate = await vm.isolates.first.load();
+    return await vm.isolates.first.load();
```

Comment:
You didn't change this, but how are we sure that the flutter isolate is always the first?
If you know how you can add a comment explaining it.

Reply:
Currently there's only ever one isolate for these tests. Originally they all just used the first isolate in-line, but I refactored it out. I named it `getFlutterIsolate` so if things changed, we could change the implementation and the name was still good. I'll add a comment explaining.

Reply:
You could use `vm.isolates.single` to assert that this is the case.

Reply:
Make sense! https://github.com/flutter/flutter/pull/19981